infrastructure/nomad/playbooks/templates/jobs/instant-bridge.nomad.j2

```diff
@@ -0,0 +1,191 @@
+          CONTRACT_DEPLOYER_KEYSTORE_PASSWORD="{{ with secret "secret/data/mev-commit" }}{{ .Data.data.contract_deployer_keystore_password }}{{ end }}"
+          {% endraw %}
+          INSTANT_BRIDGE_L1_RPC_URLS="{{ job.env['l1_rpc_urls'] }}"
+          CONTRACTS_PATH="local/contracts"
+          ARTIFACT_OUT_PATH="local"
+        EOH
+        destination = "secrets/.env"
+        env = true
+      }
+
+      template {
+        data = <<-EOH
+          #!/usr/bin/env bash
+
+          {% raw %}
+          {{- range nomadService "datadog-agent-logs-collector" }}
+            {{ if contains "tcp" .Tags }}
+          exec > >(nc {{ .Address }} {{ .Port }}) 2>&1
+            {{ end }}
+          {{- end }}
+          mkdir -p "${CONTRACT_DEPLOYER_KEYSTORE_PATH}" > /dev/null 2>&1
+          mkdir -p "${INSTANT_BRIDGE_KEYSTORE_DIR}" > /dev/null 2>&1
+            {{- with secret "secret/data/mev-commit" }}
+          CONTRACT_DEPLOYER_KEYSTORE_FILE="${CONTRACT_DEPLOYER_KEYSTORE_PATH}/${CONTRACT_DEPLOYER_KEYSTORE_FILENAME}"
+          echo '{{ .Data.data.contract_deployer_keystore }}' > "${CONTRACT_DEPLOYER_KEYSTORE_FILE}"
+          INSTANT_BRIDGE_KEYSTORE_FILE="${INSTANT_BRIDGE_KEYSTORE_DIR}/${INSTANT_BRIDGE_KEYSTORE_FILENAME}"
+          echo '{{ .Data.data.instant_bridge_keystore }}' > "${INSTANT_BRIDGE_KEYSTORE_FILE}"
+            {{ end }}
+          {% endraw %}
+
+          {% raw %}
+          {{- range nomadService "contracts-deployer" }}
+            {{ if contains "http" .Tags }}
+          URL="http://{{ .Address }}:{{ .Port }}/contracts.json"
+          CONTRACTS_FILE="/local/contracts.json"
+          curl -s -o "${CONTRACTS_FILE}" "${URL}"
+          export INSTANT_BRIDGE_SETTLEMENT_CONTRACT_ADDR="$(jq -r '.SettlementGateway' ${CONTRACTS_FILE})"
+          export INSTANT_BRIDGE_L1_CONTRACT_ADDR="$(jq -r '.L1Gateway' ${CONTRACTS_FILE})"
+            {{ end }}
+          {{- end }}
+          {% endraw %}
+
+          chmod +x local/foundry.sh && local/foundry.sh
+          chmod +x ${XDG_CONFIG_HOME}/.foundry/bin/foundryup
+          ${XDG_CONFIG_HOME}/.foundry/bin/foundryup 2>&1
+          if [ $? -ne 0 ]; then
+            echo "Failed to install foundry tools"
+            exit 1
+          fi
+          export PATH="${XDG_CONFIG_HOME}/.foundry/bin:$PATH"
+          {%- raw %}
+          {{- range nomadService "mock-l1" }}
+            {{- if contains "ws" .Tags }}
+          L1_RPC_URL="ws://{{ .Address}}:{{ .Port }}"
+            {{- end }}
+            {{- with secret "secret/data/mev-commit" }}
+          ADDRESS="$(cat "${INSTANT_BRIDGE_KEYSTORE_FILE}" | jq -r '.address')"
+            {{ end }}
+          cast send \
+            --keystore "${CONTRACT_DEPLOYER_KEYSTORE_FILE}" \
+            --password "${CONTRACT_DEPLOYER_KEYSTORE_PASSWORD}" \
+            --priority-gas-price 2000000000 \
+            --gas-price 5000000000 \
+            --value 100ether \
```

Comment:
Why is 100eth transferred here?

Reply:
This is used only when we run mock L1. So it is used to transfer initial funds to the service on L1.